scan/lib/scan/test_command_generator.rb

```diff
@@ -85,9 +85,9 @@ def pipe
 
       formatter = []
       if Scan.config[:formatter]
-        formatter << "-f `#{Scan.config[:formatter]}`"
+        formatter << "-f '#{Scan.config[:formatter]}'"
```

Comment:
Single quotes works if `:formatter` is a file path but this also needs to be backticks if `:formatter` is a gem 

I _think_ what we need to do is only use single quotes **if** `:formatter` end in `.rb` and if the file exists 🤔